utilities/private/ft_preamble_init.m

```diff
@@ -46,6 +46,20 @@
   error(msg);
 end % if nargin
 
+% check if there are fieldnames in the cfg that suggest as if the user
+% erroneously inputted a data argument
+checkdatafields = isfield(cfg, {'label' 'dimord' 'time' 'trialinfo' 'avg'});
```

Comment:
I propose to also check on "cfg". We never use cfg.cfg, but we do use data.cfg very frequently.